nn-specials/src/main/java/nl/nn/adapterframework/extensions/esb/WsdlGeneratorPipe.java

```diff
@@ -115,7 +115,7 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 			Configuration configuration = new Configuration();
 			configuration.setClassLoader(getConfigurationClassLoader());
 			adapter.setConfiguration(configuration);
-			String fileBaseName = FileUtils.getBaseName(fileName).replaceAll(" ", "_");
+			String fileBaseName = FileUtils.getBaseName(fileName).replace(" ", "_");
```

Comment:
Kan hij ook van een string een char replace maken wanneer dit mogelijk is?

Reply:
@nielsm5, ik heb er even op gezocht, maar vond geen recept die dat kan. Het kan ook alleen trouwens als het char -> char is, want anders kom je alleen op Strings uit. We kunnen daar eventueel zelf een recept voor schrijven. OpenRewrite is ook Open Source namelijk.